libs/react/src/lib/form/textarea/textarea.tsx

```diff
@@ -1,89 +1,113 @@
 import {
```

Comment:
mostly formatting of the file, also added right aligned label to form item and used classnames for class string